config/.env.local.template

```diff
@@ -21,6 +21,8 @@ REDIS_URL=redis://redis:6379/0
 
 # Celery
 # ------------------------------------------------------------------------------
+DATABASE_URL=postgis://transit_odp:transit_odp@postgres:5432/transit_odp
```

Comment:
The DATABASE_URL is generated in the /entrypoint and goesnt need to be defined here. 
Probably should go in a bash_profile or an IDE

Reply:
It was literally you who gave me this fix! Plus we don't have a bash_profile (and I'm not even using `bash` these days anyway).

I don't know what you want me to do.

Reply:
I think its best if its removed. What the caller sees here will immediately get overridden when the entrypoint is called. 
It is useful to have when running outside a container. Like for when we run `manage.py runserver_plus` Do we have a nodocker template? Maybe its best in there. Or maybe we tell the dev to add it to their environment and they can pick their best way.

Reply:
Ok, if I omit `DATABASE_URL` from `.env` and restart my containers, then I get an error saying that `DATABASE_URL` is not in the environment.

So it doesn't look like it's getting set from `/entrypoint`. Is that the bug?